src/appmixer/wiz/lib.js

```
@@ -0,0 +1,96 @@
+                'authorization': `Bearer ${context.auth.token}`
+            },
+            data
+        });
+    }
+};
+
+/**
+ * @param {array} array
+ * @returns {string}
+ */
+const toCsv = (array) => {
```

ESLint

[error] 'toCsv' is assigned a value but never used.

(no-unused-vars)